readthedocs/config/config.py

```diff
@@ -66,18 +66,12 @@
 INVALID_KEYS_COMBINATION = 'invalid-keys-combination'
 INVALID_KEY = 'invalid-key'
 
-DOCKER_DEFAULT_IMAGE = getattr(
-    settings, 'DOCKER_DEFAULT_IMAGE', 'readthedocs/build'
-)
-DOCKER_DEFAULT_VERSION = getattr(settings, 'DOCKER_DEFAULT_VERSION', '2.0')
+DOCKER_DEFAULT_IMAGE = settings.DOCKER_DEFAULT_IMAGE
+DOCKER_DEFAULT_VERSION = settings.DOCKER_DEFAULT_VERSION
 # These map to corresponding settings in the .org,
 # so they haven't been renamed.
-DOCKER_IMAGE = getattr(
-    settings,
-    'DOCKER_IMAGE',
-    '{}:{}'.format(DOCKER_DEFAULT_IMAGE, DOCKER_DEFAULT_VERSION),
-)
-DOCKER_IMAGE_SETTINGS = getattr(settings, 'DOCKER_IMAGE_SETTINGS', {})
+DOCKER_IMAGE = settings.DOCKER_IMAGE
+DOCKER_IMAGE_SETTINGS = settings.DOCKER_IMAGE_SETTINGS
```

Comment:
Why we are re-defining these here instead of using them directly from the `settings.` object in the code?

Reply:
@humitos I did not change it as it was already like this. And also they are being called in many parts of the `config.py` file. should I use them directly from the settings in the places of `config.py`?

Reply:
I vote to do that in another PR